src/logic/saveProject.js

```diff
@@ -0,0 +1,18 @@
+import {createLogic} from 'redux-logic';
+
+import {openLoginPrompt} from '../actions/ui';
+import {isUserAuthenticated} from '../selectors';
+
+import {saveCurrentProject} from './saveCurrentProject';
+
+export default createLogic({
+  type: 'SAVE_PROJECT',
```

Comment:
So I’m hoping to move us away from hard-coding action types as string constants around the codebase—one thing `redux-actions` lets us do is:

```js
export default createLogic({
  type: saveProject.toString(),
```

With `saveProject` being imported from `actions`. What do you think of this?